### PR TITLE
Portals-1490: Render entity names instead of ids

### PR DESCRIPTION
**Pull request by emmanmills**
Screen Shot 2021-03-10 at 2 00 08 PM

### CODE REVIEW

File: src/__tests__/lib/containers/SynapseTable.test.tsx [hunk 1]
```diff
@@ -606,27 +608,33 @@ describe('basic functionality', () => {
...
           <div>
             {renderTableCell({
-              entityColumnIndicies,
-              userColumnIndicies,
-              dateColumnIndicies,
-              booleanListColumnIndicies,
-              fileHandleIdColumnIndicies,
-              dateListColumnIndicies,
-              otherListColumnIndicies,
+              ...tableCellProps,
```
Comment (emmanmills) [inline src/__tests__/lib/containers/SynapseTable.test.tsx:634]: The table cell props were duplicated in several places so I refactored it.

Comment (jay-hodgson) [reply src/__tests__/lib/containers/SynapseTable.test.tsx:634]: Much cleaner, thank you.

File: src/lib/utils/SynapseClient.ts [hunk 2]
```diff
@@ -827,6 +827,22 @@ export const getEntity: GetEntity = <T>(
   ) as Promise<T>
 }
 
+/**
+ * Get a list of entity headers given by entity ids
+ * http://rest-docs.synapse.org/rest/GET/entity/type.html
+ */
+export const getEntityTypeByIds = <T extends PaginatedResults<EntityHeader>> (
```
Comment (jay-hodgson) [inline src/lib/utils/SynapseClient.ts:834]: These are not just the entity types that are being returned.  Could you rename to getEntityHeadersByIds?

File: src/lib/containers/EntityIdList.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    if (mounted && inView) {
+      getEntityTypes()
+    }
+    return () => {
+      mounted = false
+    }
+  }, [entityIdList, inView])
+
+  const getEntityTypes = async () => {
+    const entityIds = entityIdList.join(",")
```
Comment (jay-hodgson) [inline src/lib/containers/EntityIdList.tsx:28]: Prefer this transform (de-structuring) to occur in the getEntityHeadersByIds.  Could you pass the entityIdList array to that function?

File: src/lib/utils/SynapseClient.ts (hunk 2 above)
Comment (emmanmills) [reply src/lib/utils/SynapseClient.ts]: This is done.

File: src/lib/containers/EntityIdList.tsx (hunk 3 above)
Comment (emmanmills) [reply src/lib/containers/EntityIdList.tsx]: This is done.

**Pull request merged**